Source/Extensions/Blazorise.SignaturePad/wwwroot/signaturepad.js

```diff
@@ -0,0 +1,153 @@
+import { getRequiredElement } from "../Blazorise/utilities.js?v=1.2.3.0";
+import "./vendors/sigpad.js?v=1.2.3.0";
+
+const _instances = [];
+
+export function initialize(dotNetAdapter, element, elementId, options) {
+    element = getRequiredElement(element, elementId);
+
+    if (!element)
+        return;
+
+    const sigpad = new SignaturePad(element, {
+        minWidth: options.minLineWidth || 0.5,
+        maxWidth: options.maxLineWidth || 2.5,
+        backgroundColor: options.backgroundColor || "rgba(0,0,0,0)",
+        penColor: options.penColor || "black",
+        velocityFilterWeight: options.velocityFilterWeight || 0.7,
+        dotSize: options.dotSize,
+        throttle: options.throttle || 16,
+        minPointDistance: options.minPointDistance || 0.5,
+    });
+
+    if (options.value) {
+        sigpad.fromData(options.value, { clear: false });
+    }
+
+    if (options.readOnly === true) {
+        sigpad.off();
+    }
+
+    const instance = {
+        options: options,
+        sigpad: sigpad,
+    };
+
+    registerToEvents(dotNetAdapter, instance);
+
+    _instances[elementId] = instance;
+}
+
+export function destroy(element, elementId) {
+    const instances = _instances || {};
+    const instance = instances[elementId];
+
+    if (instance) {
+        if (instance.sigpad) {
+            instance.sigpad.off();
+        }
+
+        delete instances[elementId];
+    }
+}
+
+export function updateOptions(element, elementId, options) {
+    const instance = _instances[elementId];
+    if (instance && instance.sigpad && options) {
+        if (options.penColor.changed) {
+            instance.sigpad.penColor = options.penColor.value || "black";
+        }
+
+        if (options.minLineWidth.changed) {
+            instance.sigpad.minWidth = options.minLineWidth.value;
+        }
+
+        if (options.maxLineWidth.changed) {
+            instance.sigpad.maxWidth = options.maxLineWidth.value;
+        }
+
+        if (options.backgroundColor.changed) {
+            instance.sigpad.backgroundColor = options.backgroundColor.value || "rgba(0,0,0,0)";
+            const data = instance.sigpad.toData();
+            instance.sigpad.clear();
+            instance.sigpad.fromData(data);
```

Comment:
hmmm, why is this needed? Isn't there just a regular `instance.sigpad.update()`? Also how does it successfully listens to other properties and this one, not?

Reply:
Also would you even need to clear also?

Reply:
This is a use case where this was needed. Nothing else worked.

Reply:
I think I saw an example with fromdata with an option to clear. I don't know whether that's more efficient or not. or just plain equivalent. 
![image](https://github.com/Megabit/Blazorise/assets/22283161/5e1ba8eb-72ef-4c5f-a085-19ce48c8c9f3)